calico/networking/gateway-api.mdx

```diff
@@ -416,3 +418,60 @@ EOF
 ```
 
 Please note that the Gateway API CRDs will be left in place.  This is to allow for the possibility of using other Gateway API implementations in addition to the one provided by $[prodname].
+
+### Customize Gateway deployment and features
+
+The `GatewayAPI` resource has fields that allow some aspects of Gateway deployments to be customized.  For example:
+- `spec.gatewayDeployment.spec.template.metadata` allows arbitrary labels or annotations to be added to the pod that is created to implement each configured Gateway
+- `spec.gatewayDeployment.spec.template.spec.nodeSelector` allows control over where Gateway implementation pods are scheduled
+- `spec.gatewayDeployment.spec.template.spec.containers` allows control over the memory and CPU that Gateway implementation pods can use
+- `spec.gatewayControllerDeployment.spec.template.spec.nodeSelector` allows control over where the gateway controller is scheduled.
+
+For full details please see [the `GatewayAPI` reference documentation](../reference/installation/api.mdx#operator.tigera.io/v1.GatewayAPI).
+
+To make use of these customization fields, use `kubectl edit gatewayapi default` to edit the YAML for the `GatewayAPI` resource, and add or modify the customization fields that you require.
+
+### Configure additional GatewayClasses
+
+You may have a need for a tweak or customization that our `GatewayAPI` resource does not yet support.  For example, you may want to create a Gateway with EnvoyProxy config different from that which the Tigera operator provisions, and where the change that you need is not supported by an existing customization field in the `GatewayAPI` resource.
+
+To do this:
+- Extract the YAML for the EnvoyProxy and GatewayClass objects that the Tigera operator provisions.
+- Change the EnvoyProxy YAML according to your needs, and create that modified EnvoyProxy object with a different name (than "envoy-proxy-config").
+- Change the GatewayClass YAML to reference the name of the modified EnvoyProxy object, and create that modified GatewayClass with a different name (than "tigera-gateway-class").
```

Comment:
Change as above.

Reply:
Also change "filename" throughout to ``filename``.

Reply:
First point - as in my previous response.

Second point - actually these are resource names, not file names.  But OK, I think your suggestion still makes sense.